src/mock.ts

```diff
@@ -68,6 +68,11 @@ function addMockFunctionsToSchema({ schema, mocks = {}, preserveResolvers = fals
   defaultMockMap.set('String', () => 'Hello World');
   defaultMockMap.set('Boolean', () => Math.random() > 0.5);
   defaultMockMap.set('ID', () => uuid.v4());
+  defaultMockMap.set('JSON', () => ({
```

Comment:
i don't think it should be in here,
please define the mock specificly when you use JSON type.
but those are the library's default mocks, and JSON is not a default type..